atomic_reactor/plugins/post_fetch_worker_metadata.py

```diff
@@ -21,6 +27,39 @@ class FetchWorkerMetadataPlugin(PostBuildPlugin):
     key = PLUGIN_FETCH_WORKER_METADATA_KEY
     is_allowed_to_fail = False
 
+    def get_platform_metadata(self, platform, build_annotations):
+        """
+        Return the metadata for the given platform.
+        """
+
+        # retrieve all the workspace data
+        build_info = get_worker_build_info(self.workflow, platform)
+        osbs = build_info.osbs
+
+        kind = "configmap/"
+        cmlen = len(kind)
+        cm_key_tmp = build_annotations['metadata_fragment']
+        cm_frag_key = build_annotations['metadata_fragment_key']
+
+        if not cm_key_tmp or not cm_frag_key or cm_key_tmp[:cmlen] != kind:
+            self.log.warning("Bad ConfigMap annotations for platform %s", platform)
+            raise BadConfigMapError
```

Comment:
IMO this exception should be raised with the same string as the warning line above